cli/cmd/search/options.go

```diff
@@ -20,3 +20,10 @@ func init() {
 
 	flags.VarP(&opts.Query, "query", "q", "Search query terms")
 }
+
+// ResetGlobalState resets the global options state for testing.
+func ResetGlobalState() {
+	opts.Limit = 100
+	opts.Offset = 0
+	opts.Query = Query{} // Reset the query slice
+}
```

Comment:
lets avoid adding logic for tests under valid components. this can be overwritten using the CLI args